packages/cli/src/commands/list.js

```diff
@@ -5,7 +5,27 @@ import { makeRefIterator } from '@endo/daemon';
 import { withEndoHost } from '../context.js';
 import { parsePetNamePath } from '../pet-name.js';
 
-export const list = async ({ directory, follow, json }) =>
+const prettyValue = val => {
+  let result;
+  const type = typeof val;
+  if (type === 'string') {
+    result = `'${val}'`;
+  } else if (type === 'bigint') {
+    result = `${val}n`;
+  } else if (type === 'object') {
+    result = `${val}`;
+    const noise = '[object Alleged: ';
+    if (result.startsWith(noise)) {
+      result = result.substring(noise.length);
+      result = result.substring(0, result.length - 1);
+    }
```

Comment:
This might be more gracefully accomplished with `object[Symbol.toStringTag]`.

Reply:
That doesn't deal with the cases where the object in question doesn't have a string tag, plus the main motivation for me was getting rid of the "Alleged", which in spite of Mark's good intentions is in practice a noise word.